src/Apps/Home/Components/HomeNewWorksForYouRail.tsx

```diff
@@ -116,7 +116,11 @@ export const HomeNewWorksForYouRailQueryRenderer: React.FC = () => {
       environment={relayEnvironment}
       query={graphql`
         query HomeNewWorksForYouRailQuery {
-          artworksForUser(includeBackfill: true, first: 20) {
+          artworksForUser(
+            includeBackfill: true
+            first: 20
+            maxWorksPerArtist: 3
```

Comment:
+1 to the point you made in the PR desc about setting a constant! in case we gotta change this again in the future